pkgs/desktops/plasma-5/plasma-desktop/default.nix

```diff
@@ -58,7 +58,14 @@
 , runCommandLocal
 , makeWrapper
 }:
-
+let
+  # run gsettings with desktop schemas for using in kcm_accces kcm
+  # and in kaccess
+  gsettings-wrapper = runCommandLocal "gsettings-wrapper" { nativeBuildInputs = [ makeWrapper ]; } ''
+    mkdir -p $out/bin
+    makeWrapper ${glib}/bin/gsettings $out/bin/gsettings --prefix XDG_DATA_DIRS : ${gsettings-desktop-schemas.out}/share/gsettings-schemas/${gsettings-desktop-schemas.name}
```

Comment:
Is there a reason we can't just add that XDG_DATA_DIRS override directly into the wrapper for the calling application? It should then propagate down to everything else.

Reply:
There is no direct reason but we need to wrap gsettings already for kcm_access(we patch it since it is shared object) and i am reusing it in kaccess wrapper

Kaccess only needs to have schemas in gsettings invocation, so adding wrapped gsettings shouldnt be a problem

Reply:
@K900 if you think using XDG_DATA_DIRS more appropriate i can change it